### ISSUE TITLE
Some problem about training model.

### ISSUE THREAD

**Issue by Dorniwang** [TropComplique/light-head-rcnn#6]
Hi, 
    I have few questions want to learn about.
    1. How long does this code get convergence and what hardware did you use?
    2. What does the mAP can be effected by using MobileNetV2 as a backbone?
    3. I found some detiles in this code may be different with what it should be in paper, such as the batch size set to 2 images per GPU, but i found in this code it is a fixed number 1.
    4. During training, i found the loss can be low, but the validation mAP is extramly low. I wonder if this is due to the training process does not comlete?
    5. Do you have pretrained model that can offer some help?

**Comment by Dorniwang** [comment]
And another problem is that the training process is extramly slow.

Thx,
Dorni

**Comment by onzone** [comment]
Hey did you solve your issues..I am about to try training with this detector with shufflenet v2. But as per your comments it seems the training is not going properly. Have you found any other implementations?
